doc/tutorials/dnn/dnn_yolo/dnn_yolo.markdown

```diff
@@ -51,4 +51,149 @@ $ example_dnn_object_detection --config=[PATH-TO-DARKNET]/cfg/yolo.cfg --model=[
 
 @endcode
 
+
+Building a YOLO Detection Model using an ONNX Graph
+---------------------------------
+
+This guide provides a step-by-step walkthrough for running a YOLO model utilizing an ONNX graph. We focus on the YOLOX model for demonstration purposes.
+
+## Preparing the ONNX Graph
+
+### Generating the ONNX Model
+To begin, you need to generate the ONNX graph of a YOLO model. We will use the YOLOX model as an example. Detailed instructions for generating the ONNX model are available in the YOLOX [README](https://dl.opencv.org/models/yolox/README.md).
+
+Additionally, you can access a pre-trained YOLOX small ONNX graph via this [link](https://dl.opencv.org/models/yolox/yolox_s_inf_decoder.onnx).
```

Comment:
Please do not use links to dl.opencv.org, but use reference to the original repo. Also I propose to duplicate export/conversion code in tutorial.